scope-on-docker-for-mac/README.md

```diff
@@ -0,0 +1,43 @@
+### Install Docker for Mac
```

Comment:
Need some intro text for the tutorial before this heading. Explain what the user's going to see and why it's interesting to them. ("You will deploy a microservices application on your local machine, and then use the Weave Scope Probe configured to push metrics to Weave Cloud to observe your microservices app as a set of containers. Then we'll run a load test, which will allow Scope to build a live graph of the traffic between your containers.")